.github/workflows/draft-release.yml

```diff
@@ -23,34 +23,58 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Draft Release
         id: draft_release
-        uses: release-drafter/release-drafter@569eb7ee3a85817ab916c8f8ff03a5bd96c9c83e
+        uses: release-drafter/release-drafter@09c613e259eb8d4e7c81c2cb00618eb5fc4575a7 # v5.25.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838
-        with:
-          aws-region: ${{ env.AWS_REGION }}
-          role-to-assume: ${{ secrets.AWS_ACCOUNT_ROLE_ARN }}
-          role-session-name: git-api-discovery-publish-action-draft
+      - name: Set Image Tags From Release Output
+        run: |
+          cat <<HEREDOC >>${{ github.env }}
+          PRERELEASE_IMAGE_TAG=${{ steps.draft_release.outputs.tag_name }}-prerelease
+          PRERELEASE_LAMBDA_IMAGE_TAG=${{ steps.draft_release.outputs.tag_name }}-lambda-prerelease
+          HEREDOC
 
-      - name: Login to ECR and build, tag & push prerelease images
-        env:
-          PRERELEASE_IMAGE_TAG: ${{ steps.draft_release.outputs.tag_name }}-prerelease
-          PRERELEASE_LAMBDA_IMAGE_TAG: ${{ steps.draft_release.outputs.tag_name }}-lambda-prerelease
+      - name: Log In to the Container Registry
+        run: |
+           docker login ${{ env.REGISTRY }} \
+            --username ${{ github.actor }} \
+            --password-stdin <<<${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
I am not sure whether using `github.actor` here will work. 

- `github.actor` is referenced in the docs here: [Publishing Docker images](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages)

- Using the `docker login` command is documented here: [Working with the Container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic)
  - I say it's preferable to use commands rather than actions where reasonable so we don't have to deal with bumping versions of the actions.